priv/perf/apps/load_test/lib/child_chain/child_chain.ex

```diff
@@ -0,0 +1,89 @@
+# Copyright 2019-2020 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+defmodule LoadTest.ChildChain do
+  @moduledoc """
+  Utility functions for interacting with the child chain
```

Comment:
are these really utility functions to interact with the child chain? it seems like only `submit_tx` is here

Reply:
it seems the convenction is to name the module based on what it does, for example: `LoadTest.ChildChain.Deposit`